shvspy/src/dlgmountseditor.cpp

```diff
@@ -167,6 +223,65 @@ void DlgMountsEditor::listMounts()
 	m_rpcConnection->callShvMethod(rqid, aclEtcMountsNodePath(), shv::chainpack::Rpc::METH_LS);
 }
 
+void DlgMountsEditor::getMountPoint(const QString &id)
```

Comment:
toto se da v jednom callu `.broker/etc/acl/mounts/00029403001F::value()`, v tomhle prikladu to nevradi description, protoze je prazna, jinak to vrati oboji

<img width="670" alt="image" src="https://user-images.githubusercontent.com/4949019/103097228-ec1cf180-4606-11eb-8fc4-6f77bdf30351.png">